py3status/modules/volume_status.py

```diff
@@ -164,7 +164,8 @@ def toggle_mute(self):
 class PactlBackend(AudioBackend):
     def setup(self, parent):
         if self.device is None:
-            self.device = "0"
+            self.device = check_output(
+                ['pactl', 'list', 'sinks']).decode('utf-8').splitlines()[0][-1]
```

Comment:
`pactl list short sinks` makes things easier to parse.  `pactl list short sinks | cut -f 1` does it all in shell but it might be easier just to get the output and then do the 'cutting' in python.

A `# get the default device if not specified` comment or similar would be good too.

I'm not sure you have the exact same issue as #921 but this seems good to have anyway.